config/locales/en.yml

```diff
@@ -104,42 +149,131 @@ en:
+        verifications:
+          title: "Verification documents"
+          message: "In order to process your application, we will need to verify the information you provided is accurate. You will need to submit the following documents through text, email, other systems implemented by the CBO, or in-person:"
+          photo_id_accordion:
+            title: "Photo ID"
+            body: "TODO"
+          proof_of_address_accordion:
+            title: "Proof of address"
+            body: "TODO"
+          covid19_impact_accordion:
+            title: "COVID-19 impact"
+            body: "TODO"
+        application_number:
+          title: "Application Number"
+          message: "The client should write this number down. They will need it to pick up the benefit in person."
+          confirm: "Confirm that the applicant received the email or text message with the Application Number.  If not, review the contact information and make any necessary edits."
+        next_steps:
+          title: "Next Steps"
+          message_html: |
+            <ol>
+              <li>You need to submit the verification documents we discussed in the next five business days.</li>
+              <li>If we do not receive these documents, your application file will be closed and you will have to reapply.</li>
+              <li> We cannot guarantee that assistance will be available at a later date.</li>
+              <li>We will notify you once we approve your application and how it will be delivered to you.</li>
+              <li>Confirm delivery method with applicant.</li>
+            </ol>
+        receive_card:
+          title: "How will the applicant get their card?"
+          subtitle: "This can be edited later."
+          options:
+            pick_up: "Pick up"
+            deliver: "Deliver"
+            mail: "Mail"
         app_not_submitted: "Application not yet submitted."
+    confirm_messages:
+      edit:
+        title: "Confirm Activation Number Message"
+        question: "Does the applicant have their Activation Number?"
+        help: "Approving the applicatin should trigger sending the Activation Number. The applicant must have their Activation Number to use their card."
+        message_received: "Message received"
+        review: "Review contact information"
+        notification: "Application approved for <CLIENT NAME>"
+    disbursements:
+      edit:
+        title: "Complete disbursement"
+        sequence_number: "Card Sequence Number"
+        print_signature: "Print Signature Page"
+        inform_application: "Inform the applicant:"
+        activation_trouble: "If you have problems with activating the payment card, please review the instruction and contact the card issuer through the phone number or website information listed on the payment card."
+        lost_card: "If you lose the card or the card is stolen, it cannot be replaced."
+        disburse_card: "Disburse card"
```

Comment:
Do we want to add keys for "what method was the card disbursed by?" and "when was the card disbursed?"